project.sh

```diff
@@ -580,6 +584,9 @@ local-clean(){
 			fi
 			local cmd=''
 			if is-ci;then
+				if is-dry;then
+					npm-cmd install --dry-keep-package-json
```

Comment:
We know it's dry, so:
```suggestion
					dry install --dry-keep-package-json
					abort-on-error "installing"
```
And add abort-on-error (in case `dry` doesn't exist or something)